File: irshad dir/kamal/demo_python/python/L3/fact.py
#wap to find fact

num=int(input("enter a number"))
if num<0:
	print("b v+")
elif num==0 and num ==1:
	print("fact ",1)
else:
	fact=1
	for i in range(1,num+1):
		fact=fact*i
	else:
		print("fact = ",fact)

File: irshad dir/programming/java2/chap3 control statment/Fact.java
class Fact
{
public static void main(String args[])
{
int i,num,fact=1;
num=Integer.parseInt(args[0]);
for(i=1;i<=num;i++)
{
fact*=i;
}
System.out.println("the factroial of "+num+"  is "+fact);
}
}